test/token/erc1155/ImmutableERC1155.t.sol

```diff
@@ -129,6 +157,10 @@ contract ImmutableERC1155Test is Test {
         assertEq(immutableERC1155.baseURI(), "test-base-uri");
     }
 
+    function test_DeploymentShouldSetUri() public {
```

Comment:
How is this test different from line 134?

Reply:
This is one of those cases of 'you optimize what you measure'.
The forge unit test coverage tool requires a unit test to deploy the contract in order to 'mark' that bit of code as 'tested'

So this is a unit test that deploys the contract and checks that the constructor is correct - in this case, it seems like a big workaround

This unit test validates the `uri(uint256 index)` function that invokes `baseURI()` - ignoring the `index`